idp/pkg/config/defaults/defaultconfig.go

```diff
@@ -75,10 +75,11 @@ func DefaultConfig() *config.Config {
 			Scope:             "sub",
 			LoginAttribute:    "cn",
 			EmailAttribute:    "mail",
-			NameAttribute:     "sn",
+			NameAttribute:     "displayName",
 			UUIDAttribute:     "uid",
 			UUIDAttributeType: "text",
-			Filter:            "(objectClass=posixaccount)",
+			Filter:            "",
+			ObjectClass:       "posixAccount",
```

Comment:
Hm, `posixaccount` has no `displayName`, `inetOrgPerson` MAY have a `displayName`. And only `inetOrgPerson` may have a `mail` attribute. `sn` is a MUST for the STRUCTURAL `person` objectclass.

Maybe switch from `posixAccount` to `inetOrgPerson`?

```suggestion
			ObjectClass:       "inetOrgPerson",
```